features/networking/service.feature

```diff
@@ -633,3 +633,73 @@ Feature: Service related networking scenarios
       | KUBE-SEP-.+ -p tcp .* -m tcp -j DNAT --to-destination <%= cb.pod_ip %>:8080            |
       | KUBE-SERVICES -d <%= cb.service_ip %>/32 -p tcp .* -m tcp --dport 27017 -j KUBE-SVC-.+ |
       | KUBE-SVC-.+ .* -j KUBE-SEP-.+                                                          |
+
+
+  # @author jechen@redhat.com
+  # @case_id OCP-43493
+  @admin
+  @destructive
+  @4.10 @4.9
+  @vsphere-ipi @openstack-ipi @gcp-ipi @baremetal-ipi @azure-ipi @aws-ipi
+  @vsphere-upi @openstack-upi @gcp-upi @azure-upi @aws-upi
+  Scenario: Update externalIP from oc edit svc
+    Given I have a project
+    And evaluation of `project.name` is stored in the :proj_name clipboard
+    And SCC "privileged" is added to the "system:serviceaccounts:<%= project.name %>" group
+    Given I store the schedulable nodes in the :nodes clipboard
+ 
+    # Add externalIP policy to CNO
+    Given as admin I successfully merge patch resource "networks.config.openshift.io/cluster" with:
+      | {"spec":{"externalIP":{"policy":{"allowedCIDRs":["1.1.1.0/24"]}}}} |
+ 
```

Comment:
Is there a reason the network operator is not being checked after the patch request?

Reply:
@asood-rh I guess the following steps 500 seconds should be able to make sure it works. 
```
    And I wait up to 500 seconds for the steps to pass:
    """
    When I run oc create over "externalip_service1.json" replacing paths:
      | ["spec"]["externalIPs"][0] | 1.1.1.1 |
    Then the step should succeed
    """
```

Reply:
@asood-rh,  @zhaozhanqi is correct. Service can not be applied and will log the error until network operator become available again. For externalIP testing,  it normally take about 500 seconds.